### DISCUSSION TITLE
is HiKey 970 AI chip supported ?

### DISCUSSION THREAD

**Discussion by MyraBaba** [apache/incubator-mxnet#13781] [Q&A]
Is therea any experience with the Hikey 970 AI chip which almost 2TPS speed?

[a comment by an automated account is not shown]

**Comment by zachgk** [comment]
For questions, please submit them on MXNet discussion forum (https://discuss.mxnet.io), where it will get a wider audience and allow other to learn as well.
 I'm closing this issue now in favor of the discussion forum issue you will file, please feel free to re-open if closed in error.
 Thanks!

@mxnet-label-bot add [Question, ARM]
